Dockerfile

```diff
@@ -20,16 +22,18 @@ RUN touch /mailpile-data/.gnupg/docker_placeholder
 RUN chown -R mailpile:mailpile /Mailpile
 RUN chown -R mailpile:mailpile /mailpile-data
 
-# Run as non-privileged user
-USER mailpile
```

Comment:
Not running as root user is an essential security measure. A process run docker container as the root user can do many things with root rights on the host.